index.html

```diff
@@ -103,15 +103,21 @@ <h3>
         </h3>
         <pre class="idl">
           partial interface Navigator {
+            [SecureContext] boolean canShareFiles(sequence&lt;USVString&gt; file_types);
```

Comment:
Also, I don't see much advantage of allowing a sequence. As this is a synchronous check, it's better to only allow a single value. Then there is no ambiguity about which passed value is supported and which is not.

Reply:
Maybe we should generalize the name too... to `canShareType()`